test/cram/interactive_viewer_dummy.ml

```diff
@@ -4,18 +4,17 @@ let input_events : Notty.Unescape.event list ref = ref []
 let output_image : Notty.image ref = ref Notty.I.empty
 
 let open_dummy_terminal () : unit =
```

Comment:
This doesn't really make sense at the moment. It doesn't have a function yet and prevents the executable from outputting anything. Let's comment it out for now: It might or might not be useful later.